### PR TITLE
Update Spanish translation

### PR DESCRIPTION
**Pull request by RicardoRamirezR**
Fine tune Spanish translation

### CODE REVIEW

File: app/javascript/dashboard/i18n/locale/es/cannedMgmt.json [hunk 1]
```diff
@@ -67,7 +67,7 @@
       },
       "CONFIRM": {
         "TITLE": "Confirmar eliminación",
-        "MESSAGE": "¿Está seguro de eliminar ",
+        "MESSAGE": "¿Estás seguro de eliminar ",
```
Comment (pranavrajs) [inline app/javascript/dashboard/i18n/locale/es/cannedMgmt.json:70]: I believe Estás is used in informal context. I don't really know spanish, but just checking whether the usage is correct.

**Review by pranavrajs — APPROVED** [review]
@RicardoRamirezR Thanks for the PR. I believe the previous version was generated using Google Translate. This is helpful. 👍

File: app/javascript/dashboard/i18n/locale/es/cannedMgmt.json (hunk 1 above)
Comment (RicardoRamirezR) [reply app/javascript/dashboard/i18n/locale/es/cannedMgmt.json:70]: @pranavrajs mhmm, yes and not. I chose that one to be consistent across the whole translation. If you prefer the "formal" one, I can change all of them. i.e. Introduzca is the "formal" one, Introduce is the other one, etc.

Comment (pranavrajs) [reply app/javascript/dashboard/i18n/locale/es/cannedMgmt.json:70]: @RicardoRamirezR Thanks for the details. I would suggest to use the formal one if that is possible. :)

**Review by pranavrajs — CHANGES_REQUESTED** [review]
As discussed, we would go ahead with the formal language :)

**Comment by RicardoRamirezR** [comment]
@pranavrajs done the formal one!

File: app/javascript/dashboard/i18n/locale/es/cannedMgmt.json (hunk 1 above)
Comment (RicardoRamirezR) [reply app/javascript/dashboard/i18n/locale/es/cannedMgmt.json]: Done

**Pull request merged**